livedata/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -519,6 +525,9 @@ internal class ChannelControllerImpl(
                 upsertMessage(processedMessage)
                 Result(processedMessage, null)
             } else {
+
+                logger.logE("Failed to send message with id ${newMessage.id} and text ${newMessage.text}", result.error())
```

Comment:
@JcMinarro is this the best way to log an error? I noticed on the sample app that messages sometimes fail to send. It only happens on my real device though, can't replicate it on the emulator. So how do we log an error and have it show up in crashlytics/firebase?

Reply:
With the current implementation we have we are not sending any log to Firebase, only crashes